plugins/AccountStats.py

```diff
@@ -0,0 +1,124 @@
+# coding=utf-8
+from datetime import date
+
+from plugins.Plugin import Plugin
+import modules.Poloniex as Poloniex
+import sqlite3
+
+BITCOIN_GENESIS_BLOCK_DATE = "2009-01-03 18:15:05"
+DAY_IN_SEC = 86400
+DB_DROP = "DROP TABLE IF EXISTS history"
+DB_CREATE = "CREATE TABLE IF NOT EXISTS history(" \
+            "id INTEGER UNIQUE, open TIMESTAMP, close TIMESTAMP," \
+            " duration NUMBER, interest NUMBER, rate NUMBER," \
+            " currency TEXT, amount NUMBER, earned NUMBER, fee NUMBER )"
+DB_INSERT = "INSERT OR REPLACE INTO 'history'" \
+            "('id','open','close','duration','interest','rate','currency','amount','earned','fee')" \
+            " VALUES (?,?,?,?,?,?,?,?,?,?);"
+DB_GET_LAST_TIMESTAMP = "SELECT max(close) as last_timestamp FROM 'history'"
+DB_GET_FIRST_TIMESTAMP = "SELECT min(close) as first_timestamp FROM 'history'"
+DB_GET_TOTAL_EARNED = "SELECT sum(earned) as total_earned, currency FROM 'history' GROUP BY currency"
+DB_GET_24HR_EARNED = "SELECT sum(earned) as total_earned, currency FROM 'history' " \
+                     "WHERE close BETWEEN datetime('now','-1 day') AND datetime('now') GROUP BY currency"
+
+
+class AccountStats(Plugin):
+    last_notification = 0
+
+    def on_bot_init(self):
+        super(AccountStats, self).on_bot_init()
+        self.init_db()
+
+    def after_lending(self):
+        self.update_history()
+        self.notify_daily()
+
+    # noinspection PyAttributeOutsideInit
+    def init_db(self):
+        self.db = sqlite3.connect(r'market_data\loan_history.sqlite3')
```

Comment:
That `r` looks like a typo, can't find reference to it in the docs.

Unless it means "relative" to current directory?